### ISSUE TITLE
A.04.IR16.IR17.link.to.get.harmonised.service.metadata

### ISSUE THREAD

**Issue by jensscheerlinck** [inspire-eu-validation/ats-harmonised-sds#2]
_This issue has been extracted from the issue list on:https://ies-svn.jrc.ec.europa.eu/issues/2685_
# Comment

The Prerequisite does not make sense because:
- it puts a condition on the metadata document that will be fetched during the test.The condition it describes should be moved to the Test Method.
  -it assumes that what comes back is an ISO 19139 document while it could be a Capabilities document.
  The 'equivalent' for Network Services is the Get Network Service Metadata Response document which is often an OGC Capabilities document.

**Comment by cportele** [comment]
These SDS-related conformance classes / ATSs have not been part of the review: https://ies-svn.jrc.ec.europa.eu/issues/2685

Proposed resolution:
- Keep the issue open and revisit as part of the work on the SDS ATS.

**Comment by cportele** [comment]
Changed to rejected and wontfix as this reflects the status during this review. The issue may be revisited once the SDS ATS is ready for review.
